brainkeeper/package.json

```diff
@@ -15,13 +15,13 @@
     "@angular/cdk": "~8.2.3",
     "@angular/common": "~8.2.9",
     "@angular/compiler": "~8.2.9",
-    "@angular/core": "~8.2.9",
+    "@angular/core": "^8.2.14",
     "@angular/forms": "~8.2.9",
     "@angular/material": "^8.2.3",
     "@angular/platform-browser": "~8.2.9",
     "@angular/platform-browser-dynamic": "~8.2.9",
-    "@angular/router": "~8.2.9",
-    "dexie": "^2.0.4",
```

Comment:
Please readd this line, the dexie depencency is still needed for the database service